docs/projects/ad7616_sdz/index.rst

```diff
@@ -0,0 +1,336 @@
+     - 0
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Software GPIO number is calculated as follows:
+
+-  Zynq-7000: if PS7 is used, then offset is 54
+
+============= ============= ================
+GPIO signal   HDL GPIO EMIO Software GPIO nb
+============= ============= ================
+adc_reset_n   43            97
+adc_hw_rngsel 42:41         96:95
+adc_os **     40:38         94:92
+adc_seq_en    37            91
+adc_burst **  36            90
+adc_chsel     35:33         89:87
+adc_crcen **  32            86
+============= ============= ================
+
+.. note::
+
+   Legend
+     
+   -   ``**`` - only for SER_PAR_N=1 (serial interface)
+            
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+========================  ==========
+Instance                  Address
+========================  ==========
+axi_ad7616_dma            0x44a30000
+ad7616_pwm_gen            0x44b00000
+spi_ad7616_axi_regmap **  0x44a00000
+axi_ad7616 *              0x44a80000
+========================  ==========
+
+.. note::
+
+   Legend
+     
+   -   ``*`` - instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` - instantiated only for SER_PAR_N=1 (serial interface)
+
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+=============== === ========== ===========
+Instance name   HDL Linux Zynq Actual Zynq
+=============== === ========== ===========
+axi_ad7616_dma  13  57         89
+spi_ad7616 **   12  56         88
+axi_ad7616 *    10  54         87
+=============== === ========== ===========
+
+.. note::
+
+   Legend
+     
+   -   ``*`` - instantiated only for SER_PAR_N=0 (parallel interface)
+   -   ``**`` - instantiated only for SER_PAR_N=1 (serial interface)
+
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+Setup guide
+-------------------------------------------------------------------------------
+
+Below is a user guide which help you start with your setup.
+
+-  :dokuwiki:`AD7616_SDZ (AD7616_SDZ) <resources/eval/user-guides/ad7616-sdz>`
+
+Connections and hardware changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   **The following hardware changes are required:**
+ 
+   (**Please note:** Because of the **SDP-I-FMC** the level of the **VADJ** in 
+   the carrier board needs to be set to **3.3V**.
+     
+   Depending on the required interface mode, some hardware modifications need to 
+   be done.
+   
+   -   **SL5** - unmounted - Parallel interface
+   -   **SL5** - mounted - Serial interface
+
+Resources
+-------------------------------------------------------------------------------
+
+-  :git-hdl:`ad7616_sdz HDL project <master:projects/ad7616_sdz>`
```

Comment:
These links should be in the `HDL related` section

Reply:
Done.